cf/api/domains_test.go

```diff
@@ -306,23 +306,46 @@ var _ = Describe("DomainRepository", func() {
 				config.SetApiVersion("2.2.0")
 			})
 
-			It("uses the shared domains endpoint", func() {
-				setupTestServer(
-					testapi.NewCloudControllerTestRequest(testnet.TestRequest{
-						Method:  "POST",
-						Path:    "/v2/shared_domains",
-						Matcher: testnet.RequestBodyMatcher(`{"name":"example.com", "wildcard": true}`),
-						Response: testnet.TestResponse{Status: http.StatusCreated, Body: `
+			Context("when router_group_guid is the empty string", func() {
```

Comment:
This empty context should be removed. See https://github.com/cloudfoundry/cli/wiki/Coding-Style-Guide#general-test-structure.